src/app/auth/auth-login/auth-login.component.ts

```diff
@@ -60,54 +69,57 @@ export class AuthLoginComponent implements OnInit {
       this.newRelic.setCustomAttribute('login status', message);
     });
     return this.authService.login({
-      email: this.loginForm.value.email,
+      username: this.loginForm.value.username,
       password: this.loginForm.value.password,
     }).subscribe(
-      res => {
-        nrLoginTracer('login successful');
-        this.newRelic.actionText('login successful');
-        return this._handleNavigation(res.programs);
+      globalRes => {
+        return this.authService.directLoginWithApikey({
```

Comment:
the logic here is still incorrect. After we get the stacks, we store them in local storage and redirect user to experience switcher page. Then we need to make `auth` requests for each stacks to get the experiences and gather them together.